src/device_trezor/trezor/transport.cpp

```diff
@@ -56,6 +56,11 @@ namespace trezor{
     return true;
   }
 
+  bool t_serialize(const epee::wipeable_string & in, std::string & out){
+    out.assign(in.data(), in.size());
+    return true;
+  }
```

Comment:
Does the serializtion buffer get wiped later ?

Reply:
Good question! Yes, the caller is responsible for wiping it after usage. 
It is used only here: https://github.com/monero-project/monero/blob/62ca79d96aa3e941f7faa53fc1c78b03bc1a19f7/src/device_trezor/trezor/transport.hpp#L83

The scope leave handler wipes it then.